Fabric.Terminology.API/Program.cs

```diff
@@ -19,7 +19,8 @@ public static void Main()
                 .ConfigureAppConfiguration((context, config) =>
                     {
                         config.Sources.Clear();
-                        config.AddJsonFile("appsettings.json", optional: false);
+                        config.AddJsonFile("appsettings.json", optional: false)
```

Comment:
FYI - in the Release off of the CI Build, we use build variables to swap these out based on the deploy environment.  It should still replace values in the appsettings.json and I do have each value that you are setting up to pattern match defined ... so I think we'll be good - but if something breaks, this would be a good first place to start.

Reply:
I don't see any replacement happening, am I missing something? https://healthcatalyst.visualstudio.com/Clinical%20Decision/_build/results?buildId=63260&_a=summary

I don't care if we have a `dev` version of appsettings.json or if we do it through the build.  I can abandon this pr if you feel strongly for using the build for replacement